### PR TITLE
Add support for Execution Environments

### PR DESCRIPTION
**Pull request by mayaCostantini**
- [x] Add support for Execution Environments
- [x] Add a `.gitignore`
- [x] Increase timeout for connecting to Keycloak and Rekor

### CODE REVIEW

File: roles/sigstore_scaffolding/tasks/podman/tuf.yml [hunk 1]
```diff
@@ -23,8 +23,8 @@
     validate_certs: false
     return_content: true
   until: "rekor_public_key_result.status == 200"
-  retries: 5
```
Comment (sabre1041) [inline roles/sigstore_scaffolding/tasks/podman/tuf.yml:26]: Do we want to make these and the others configurable as different operating environment will have different failure rates (time to pull down the image etc)

File: README.md [hunk 2]
```diff
@@ -123,6 +123,30 @@ cosign verify --rekor-url=$REKOR_URL <image>
...
+
+```
+---
+base_hostname: base_hostname
+```
+
+5. Run the deployment job inside the Execution Environment:
+
+```
+ansible-runner run -i inventory -p playbooks/install.yml --container-image=my_ee .
```
Comment (sabre1041) [inline README.md:147]: We should be using [ansible-navigator](https://ansible-navigator.readthedocs.io) instead of calling `ansible-runner` directly

**Review by sabre1041 — CHANGES_REQUESTED** [review]
A couple of comments

**Comment by mayaCostantini** [comment]
Thank you for the review @sabre1041, adjustements were made in https://github.com/sabre1041/sigstore-ansible/pull/2/commits/b258850ade584b66cfd30347cf3e9d1aea30a71b.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**